Tasks/Common/nuget-task-common/package.json

```diff
@@ -16,8 +16,8 @@
     "ltx": "^2.6.2",
     "mockery": "^1.7.0",
     "vso-node-api": "^5.0.5",
-    "vsts-task-lib": "2.0.2-preview",
-    "vsts-task-tool-lib": "0.4.0",
+    "vsts-task-lib": "^2.0.6",
```

Comment:
i would recommend pinning to a specific version so the version doesnt slide while a release is stabilizing

Reply:
That was intentional.  I am trying to strike a balance between picking up framework changes, like proxy support, without requiring users to update their build definitions.  I am relying on our test framework to ferret out bugs in minor releases before we go live.

Reply:
Based on our conversation I've pinned versions...